data/base-spells.yaml

```diff
@@ -1129,9 +1129,10 @@ spell_data:
     mana: 15
     recast: 1
   Magnetic Ballista:
-    skill: Targeted Magic
+    skill: Utility
```

Comment:
❓ Isn't MAB both TM and Utility? If so, why the change?

Reply:
Probably so that it will recast on the expire message.